.github/workflows/linters.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Setup Webhint
```

Comment:
Please add `.eslintrc.json`, `.hintrc`, and `.stylelintrc.json` files from config as well. Kindly use the same [config](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

Reply:
I have these files on my local repo, what could be the reason they are not being pushed?